src/colvarmodule.cpp

```diff
@@ -254,6 +254,7 @@ int colvarmodule::parse_global_params(std::string const &conf)
     std::string index_file_name;
     size_t pos = 0;
     while (parse->key_lookup(conf, "indexFile", &index_file_name, &pos)) {
+      cvm::log("# indexFile = \""+index_file_name+"\"\n");
```

Comment:
This feels like it could be made more natural with the improved parser - that is, allowing repeated calls to get_keyval ?

Reply:
Definitely yes.  I plan on implementing that separately.